api/posters/statistics/send.spec.ts

```diff
@@ -0,0 +1,98 @@
+import {PosterImageSizes, PosterSteps} from '@nebula/types/poster'
+
+import PosterModel from '../poster.model'
+
+import * as helpers from './helpers'
+import {sendPosterStatistics} from './send'
+
+const posterImages: PosterImageSizes = {
+  twitter: 'twitter.png',
+  highQualityPoster: 'high-quality.png',
+  instagram: 'instagram.png',
+  openGraph: 'openGraph.png',
+}
+
+const posterSlug = 'slug'
+
+describe('sendPosterStatistics', () => {
+  afterEach(jest.clearAllMocks)
+
+  describe("when there's no posters in the current week", () => {
+    it("sends an email with 'No new posters this week'", async () => {
+      /**
+       * Make sure there's no posters in the database due to race conditions in the DB.
+       */
+      const posters = await PosterModel.scan().exec()
+      posters.forEach(async poster => {
+        await poster.delete()
+      })
+      const sendEmailSpy = jest
+        .spyOn(helpers, 'sendEmail')
+        .mockResolvedValueOnce()
+      await sendPosterStatistics()
+      expect(sendEmailSpy).toHaveBeenCalledWith(
+        expect.stringContaining('No new posters this week'),
```

Comment:
We can also use `toMatchInlineSnapshots` here to make sure the email is consistent.